.github/files/supressions_list.txt

```diff
@@ -0,0 +1 @@
+memleak:feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/uci_helper.c:372
```

Comment:
This memaleak is actually present. The unitialized warning in the ubus plugin of hostapd was invalid